### PR TITLE
Fixed the cherry-pick of changes to FocusTrapZone

### PR DESCRIPTION
**Pull request by ThomasMichon**
The previous cherry-pick applied the change to the wrong part of `FocusTrapZone`.

### CODE REVIEW

**Comment by fabricteam** [comment]
## [Perf Analysis](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">7.0 Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>BaseButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23896/perf-test/BaseButton-mount_base.html">708</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23896/perf-test/BaseButton-mount.html">658</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Breadcrumb</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23896/perf-test/Breadcrumb-mount_base.html">36153</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23896/perf-test/Breadcrumb-mount.html">34633</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Checkbox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23896/perf-test/Checkbox-mount_base.html">1345</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23896/perf-test/Checkbox-mount.html">1344</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>CheckboxBase</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23896/perf-test/CheckboxBase-mount_base.html">1075</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23896/perf-test/CheckboxBase-mount.html">1080</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ChoiceGroup</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23896/perf-test/ChoiceGroup-mount_base.html">4155</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23896/perf-test/ChoiceGroup-mount.html">4175</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ComboBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23896/perf-test/ComboBox-mount_base.html">792</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23896/perf-test/ComboBox-mount.html">748</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>CommandBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23896/perf-test/CommandBar-mount_base.html">6665</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23896/perf-test/CommandBar-mount.html">6266</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>ContextualMenu</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23896/perf-test/ContextualMenu-mount_base.html">11990</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23896/perf-test/ContextualMenu-mount.html">11242</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DefaultButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23896/perf-test/DefaultButton-mount_base.html">955</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23896/perf-test/DefaultButton-mount.html">905</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRow</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23896/perf-test/DetailsRow-mount_base.html">3160</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23896/perf-test/DetailsRow-mount.html">3178</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowFast</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23896/perf-test/DetailsRowFast-mount_base.html">2854</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23896/perf-test/DetailsRowFast-mount.html">3133</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowNoStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23896/perf-test/DetailsRowNoStyles-mount_base.html">2979</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23896/perf-test/DetailsRowNoStyles-mount.html">3002</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Dialog</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23896/perf-test/Dialog-mount_base.html">1873</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23896/perf-test/Dialog-mount.html">1918</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DocumentCardTitle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23896/perf-test/DocumentCardTitle-mount_base.html">1546</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23896/perf-test/DocumentCardTitle-mount.html">1539</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Dropdown</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23896/perf-test/Dropdown-mount_base.html">2176</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23896/perf-test/Dropdown-mount.html">2207</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusTrapZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23896/perf-test/FocusTrapZone-mount_base.html">1493</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23896/perf-test/FocusTrapZone-mount.html">1502</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23896/perf-test/FocusZone-mount_base.html">1542</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23896/perf-test/FocusZone-mount.html">1557</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>IconButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23896/perf-test/IconButton-mount_base.html">1460</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23896/perf-test/IconButton-mount.html">1484</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Label</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23896/perf-test/Label-mount_base.html">258</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23896/perf-test/Label-mount.html">286</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Layer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23896/perf-test/Layer-mount_base.html">1645</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23896/perf-test/Layer-mount.html">1648</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Link</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23896/perf-test/Link-mount_base.html">381</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23896/perf-test/Link-mount.html">373</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MenuButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23896/perf-test/MenuButton-mount_base.html">1208</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23896/perf-test/MenuButton-mount.html">1232</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MessageBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23896/perf-test/MessageBar-mount_base.html">1693</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23896/perf-test/MessageBar-mount.html">1786</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Nav</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23896/perf-test/Nav-mount_base.html">2544</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23896/perf-test/Nav-mount.html">2700</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>OverflowSet</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23896/perf-test/OverflowSet-mount_base.html">1216</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23896/perf-test/OverflowSet-mount.html">1083</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Panel</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23896/perf-test/Panel-mount_base.html">1140</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23896/perf-test/Panel-mount.html">1278</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23896/perf-test/Persona-mount_base.html">805</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23896/perf-test/Persona-mount.html">824</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Pivot</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23896/perf-test/Pivot-mount_base.html">1248</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23896/perf-test/Pivot-mount.html">1242</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>PrimaryButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23896/perf-test/PrimaryButton-mount_base.html">1090</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23896/perf-test/PrimaryButton-mount.html">1087</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Rating</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23896/perf-test/Rating-mount_base.html">6208</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23896/perf-test/Rating-mount.html">6399</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SearchBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23896/perf-test/SearchBox-mount_base.html">984</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23896/perf-test/SearchBox-mount.html">967</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Shimmer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23896/perf-test/Shimmer-mount_base.html">2059</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23896/perf-test/Shimmer-mount.html">2090</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Slider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23896/perf-test/Slider-mount_base.html">1141</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23896/perf-test/Slider-mount.html">1292</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23896/perf-test/SpinButton-mount_base.html">4310</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23896/perf-test/SpinButton-mount.html">4671</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Spinner</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23896/perf-test/Spinner-mount_base.html">345</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23896/perf-test/Spinner-mount.html">352</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SplitButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23896/perf-test/SplitButton-mount_base.html">2582</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23896/perf-test/SplitButton-mount.html">2587</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Stack</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23896/perf-test/Stack-mount_base.html">413</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23896/perf-test/Stack-mount.html">413</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithIntrinsicChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23896/perf-test/StackWithIntrinsicChildren-mount_base.html">1310</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23896/perf-test/StackWithIntrinsicChildren-mount.html">1325</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithTextChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23896/perf-test/StackWithTextChildren-mount_base.html">3910</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23896/perf-test/StackWithTextChildren-mount.html">4008</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SwatchColorPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23896/perf-test/SwatchColorPicker-mount_base.html">8799</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23896/perf-test/SwatchColorPicker-mount.html">9086</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TagPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23896/perf-test/TagPicker-mount_base.html">2477</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23896/perf-test/TagPicker-mount.html">2503</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TeachingBubble</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23896/perf-test/TeachingBubble-mount_base.html">42969</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23896/perf-test/TeachingBubble-mount.html">41815</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Text</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23896/perf-test/Text-mount_base.html">366</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23896/perf-test/Text-mount.html">363</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TextField</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23896/perf-test/TextField-mount_base.html">1176</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23896/perf-test/TextField-mount.html">1174</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Toggle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23896/perf-test/Toggle-mount_base.html">670</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23896/perf-test/Toggle-mount.html">689</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>button</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23896/perf-test/button-mount_base.html">86</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/23896/perf-test/button-mount.html">91</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9423"/>

File: change/office-ui-fabric-react-e807606c-faa7-44a3-bba7-d08596317c4f.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "Fix cherry-pick of FocusTrapZone changes",
```
Comment (khmakoto) [inline change/office-ui-fabric-react-e807606c-faa7-44a3-bba7-d08596317c4f.json:3]:
```suggestion
  "comment": "fix: Fixing previous incorrect implementation of iframe handling in FocusTrapZone.",
```

**Review by khmakoto — APPROVED** [review]

**Comment by khmakoto** [comment]
Closing and reopening to trigger a merge with master.

**Pull request merged**